app/services/ajax.js

```diff
@@ -62,24 +62,12 @@ export default Ember.Service.extend({
       options.contentType = options.contentType || 'application/json; charset=utf-8';
     }
     success = options.success || (function() {});
-    options.success = (data, status, xhr) => {
```

Comment:
I've checked the API and unfortunately there's a bug that may prevent us from implementing this, so for now I'd leave `loadFlashes` here.